test/controllers/update/update.statement.validation.errors.controller.spec.ts

```diff
@@ -177,6 +179,41 @@ describe("Update statement validation errors controller", () => {
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
     });
 
+    test(`in a change journey, renders the Update statement validation errors page when there are no added or ceased BOs`, async () => {
+      mockHasValidStatements.mockImplementation((req: Request, __: Response, next: NextFunction) => {
+        req['statementErrorList'] = ['There is at least one active registrable beneficial owner.'];
+        next();
+      });
+      const BOI_MOCK: beneficialOwnerIndividualType.BeneficialOwnerIndividual = BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK;
+      BOI_MOCK.ceased_date = undefined;
+      BOI_MOCK.ch_reference = "test";
+      console.log("TESST " + BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK);
+      mockGetApplicationData.mockReturnValue({
+        entity_name: 'Potato',
+        entity_number: 'OE991992',
+        [beneficialOwnerIndividualType.BeneficialOwnerIndividualKey]: [ BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK ],
+        beneficial_owners_statement: 'ALL_IDENTIFIED_ALL_DETAILS',
+        update: {
+          no_change: false,
+          registrable_beneficial_owner: 1,
+        },
+      });
+
+      const resp = await request(app).get(UPDATE_STATEMENT_VALIDATION_ERRORS_URL);
+
+      expect(resp.status).toEqual(200);
+
+      expect(resp.text).toContain('The statements you&#39;ve chosen do not match the information provided in this update');
+      expect(resp.text).toContain(UPDATE_REGISTRABLE_BENEFICIAL_OWNER_URL);
+      expect(resp.text).toContain('Potato - OE991992');
+      expect(resp.text).toContain('There is at least one active registrable beneficial owner.');
+      expect(resp.text).toContain('The entity has reasonable cause to believe that at least one person has become or ceased to be a registrable beneficial owner during the update period.');
+
+      expect(resp.text).not.toContain('value="statement-resolution-change-information" checked');
+      expect(resp.text).not.toContain('value="statement-resolution-change-statement" checked');
+      expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+    });
+
```

Comment:
left this comment on Christy's PR as well:

"

this test file keeps growing and it is a lot of repeated checks - are these tests really necessary compared to ones that are already there? I don't think we need to have separate tests in here for every scenario because your tests in test/middleware/statement.validation.middleware.spec.ts cover all the error scenarios.

You could just add your error message into req['statementErrorList'] of an existing test but even then not sure what value you're getting out of that.
"